### PR TITLE
Fix for PMMP api 3.0.0-ALPHA7

### PR DESCRIPTION
**Pull request by GamakCZ**
### Purpose and Effect of This Pull Request

Plugins from PocketMine will compatible with GenisysPro

### CODE REVIEW

**Comment by DerddyDert** [comment]
Thanks. This will be reviewed today.

**Comment by MasterOfTheRealm** [comment]
@GamakCZ tested?

File: src/pocketmine/command/CommandExecutor.php [hunk 1]
```diff
@@ -28,10 +28,10 @@
 	 * @param CommandSender $sender
 	 * @param Command       $command
 	 * @param string        $label
-	 * @param string[]      $args
```
Comment (Leo3418) [inline src/pocketmine/command/CommandExecutor.php]: Effect of this change is unknown, since PocketMine-MP doesn't have this line modified.

Comment (DerddyDert) [inline src/pocketmine/command/CommandExecutor.php]: @GamakCZ when you get a chance please follow up to the responses. Thanks

**Comment by armagadon159753** [comment]
I tested this but the server crash after change because of the plugon even after their modifications

**Comment by MCNationPE** [comment]
test without plugins

**Comment by MCNationPE** [comment]
Look this code
`public function onCommand(CommandSender $sender, Command $command, string $label, array $args):bool;`

**Comment by armagadon159753** [comment]
Yes @MCNationPE no crash without plugins but I made the right modification for the plugin tester

File: src/pocketmine/command/CommandExecutor.php (hunk 1 above)
Comment (GamakCZ) [inline src/pocketmine/command/CommandExecutor.php]: It does not need to be changed.

Comment (DerddyDert) [inline src/pocketmine/command/CommandExecutor.php]: Close pr if you don't want to add it. Or let us know what you want to do.

Comment (GamakCZ) [inline src/pocketmine/command/CommandExecutor.php]: I mean line 31

Comment (DerddyDert) [inline src/pocketmine/command/CommandExecutor.php]: @Frago9876543210

**Comment by DerddyDert** [comment]
@GamakCZ  I'm closing this. You should make an "api 3.0.0-ALPHA7" branch

**Comment by GamakCZ** [comment]
I have not permissions to make branch

**Pull request closed without merging**